test/wpackettest.c

```diff
@@ -26,6 +26,24 @@ static const unsigned char simpleder[] = {
     0xfc, 0x04, 0x00, 0x01, 0x02, 0x03, 0xff, 0xfe, 0xfd
 };
 
+static const unsigned char quic1[] = { 0x80, 0x00, 0x00, 0x01, 0x09 };
```

Comment:
It would be useful to have the comment about what each of these represent included here rather than just in the test_WPACKET_quic area - i.e. both places helps keep the context clear IMHO